components/icons/EllipseIconGreen.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
A gente não precisa mais desse arquivo, correto?
Lembra de apagar ;)

Reply:
inclusive nao precisa nem no react normal mais https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj

Reply:
Feito!